### PR TITLE
Added Sumologic-control service

### PR DESCRIPTION
**Pull request by iderdik**
Separates control/proxy tier logs from workers for easier searching by app name.

### CODE REVIEW

**Comment by chr0n1x** [comment]
:+1:

File: v1/fleet_units/sumologic-control.service [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    -e SUMO_COLLECTOR_NAME=mesos_app_collector \
+    -e SUMO_ACCESS_ID=`etcdctl get /sumologic_id` \
+    -e SUMO_ACCESS_KEY=`etcdctl get /sumologic_secret` \
+    behance/docker-sumologic"
+ExecStop=/usr/bin/docker stop sumologic-control
+
+[X-Fleet]
+Global=true
+MachineMetadata=role=control
+MachineMetadata=role=proxy
```
Comment (nBerg) [inline v1/fleet_units/sumologic-control.service:29]: Should the regular one be modified to run only on workers?

File: v1/fleet_units/sumologic-control.service [hunk 2]
```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Sumologic Collector
```
Comment (chr0n1x) [inline v1/fleet_units/sumologic-control.service]: maybe update this description though

File: v1/fleet_units/sumologic-control.service (hunk 1 above)
Comment (iderdik) [inline v1/fleet_units/sumologic-control.service:29]: Yep, I had the change in the file unsaved :cold_sweat:

**Pull request merged**